build-system/tasks/e2e/mocha-karma-mimic-reporter.js

```diff
@@ -17,15 +17,15 @@
 
 const {Base} = require('mocha').reporters;
 const {inherits} = require('mocha').utils;
-const {reportTestFinished} = require('./report-test-status');
-const {symbols} = require('./karma.conf').mochaReporter;
+const {reportTestFinished} = require('../report-test-status');
+const {symbols} = require('../karma.conf').mochaReporter;
 
 /**
  * Custom Mocha reporter for CI builds.
  * Mimics the style of the Karma reporter on Travis.
  * @param {*} runner
  */
-function ciReporter(runner) {
+function KarmaMimicReporter(runner) {
```

Comment:
Let's call this MochaDotsReporter instead

Reply:
Done.